live-build/misc/ansible-roles/appliance-build.minimal-common/tasks/main.yml

```diff
@@ -41,12 +41,14 @@
     comment: Delphix User
     home: /export/home/delphix
     password:
-      "{{ lookup('env', 'APPLIANCE_PASSWORD') | password_hash('sha512') }}"
+      "$6$oSDBhe8aMbPzcuGf$Zpnq.v8h6hdq/eYXs61oxvCK/b08mx.5uD61xoJq8SrQS\
```

Comment:
do we really want to make these public? they are now...

should we create a new environment variable instead? e.g. `APPLIANCE_PARTNER_PASSWORD`?

Reply:
I do not intent to merge this PR. This is just a special build that won't ever make it into our repo.

Reply:
I don't see the need to change this line of code at all, actually. If our goal is to do a single manual build, we can set APPLIANCE_PASSWORD to whatever we want in the environment of that build.